metrics/serializers.py

```diff
@@ -0,0 +1,159 @@
+from rest_framework import serializers
+from .models import *
+
+
+class SensorTypeSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = SensorType
+        fields = ['id', 'name']
+
+
+class UnitSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Unit
+        fields = ['id', 'name']
+
+
+class SensorListSerializer(serializers.ModelSerializer):
+    """ GET --> just a list of all sensors"""
+
+    sensors_count = serializers.SerializerMethodField()
+
+    def get_sensors_count(self, obj):
+        return Sensor.objects.all().count()
```

Comment:
DRF's built-in pagination has the count key and you don't need to sacrifice code's performance, maintainability, and so other things

Reply:
fixed